src/storage/memory_store.ts

```diff
@@ -188,6 +188,13 @@ export class MemoryStore implements IStore {
     // This is a noop since the memory store always starts clean.
   }
 
+  /**
+   * Prepare the store for history processing.
+   */
+  async preProcess(): Promise<void> {
```

Comment:
Should we rename it to `preProcessHistory()`?

Reply:
Done